test/br/com/caelum/iogi/vraptor/OgnlParametersProviderTestVRaptorTests.java

```diff
@@ -125,4 +125,28 @@ public void removeFromTheCollectionIfAnElementIsCreatedWithinACollectionButNoFie
         assertThat(house.owners, hasSize(1));
         assertThat(house.owners.get(0), is(equalTo("guilherme")));
     }
+	
+	static class MyClass {
+		private String name;
+
+		public String getName() {
+			return name;
+		}
+
+		public void setName(String name) {
+			this.name = name;
+		}
+		
+		public void setEmptyProperty() {
+			
+		}
+	}
+	
+	@Test
+	public void setterWithoutProperty() {
```

Comment:
i'd just rename this test, for `shouldIgnoreMethodsWithSetPrefix` or something like that
what do you think @dtelaroli? besides that, everything looks great. Thank you for your pr